modules/swagger-codegen/src/main/resources/aspnetcore/OPS.md.mustache

```diff
@@ -0,0 +1,25 @@
+```
+docker run --rm 
+-v D:/swagger/s:/local 
+jixiu/codegen:lasted generate 
+-i /local/user.yaml 
+-c /local/server.json 
+-l aspnetcore 
+-o /local 
+-t /local/resources/aspnetcore 
+-s 
+```
+
+-v ������docker���ļ���ӳ���ϵ
```

Comment:
@seven1986 instead of providing Chinese instruction, what about using English instead?

I can read simplified Chinese but I believe our users would prefer English :)